### ISSUE TITLE
make as discord bot

### ISSUE THREAD

**Issue by phoebe-leong** [Theblockbuster1/disboard-auto-bump#18]
I read a bit of the README and if I read correctly, it states that the user has to keep the tab open all the time. Wouldn't this get annoying?
So I'm suggesting to either make this repository (or fork it) into a discord bot where the owners of the bot can simply enter in their discord server invite (or something along those lines, maybe this isn't possible at all) and the bot continuously bumps the server.
Do you think this could be done?

**Comment by Theblockbuster1** [comment]
Yes, it could potentially get annoying, but you could just set the tab running, pin it, and leave it. However, putting this in a Discord bot isn't a bad of an idea, it potentially could use a library such as puppeteer to run through a bot.

I'm not sure if I currently have the time to implement it, but it definitely seems like something I should consider! Feel free to fork and try yourself, or otherwise I'll get to doing it myself when I find time to do so.

**Comment by Dxniel0** [comment]
I think the bot on discord is a great idea

**Comment by thedarklord843** [comment]
interesting idea

**Comment by godtengri** [comment]
Doing this will ban your server on DISBOARD for auto-bumping.

**Comment by Theblockbuster1** [comment]
@godtengri How would it put the user in any more risk of getting banned than using this browser extension? I think there is no difference; you are equally as likely to get banned using this extension, as you are to get banned using the Discord bot (which will basically be running this extension inside through Chrome).

**Comment by phoebe-leong** [comment]
hey maybe to get around that possibility people could put a limit on the bumps per second or smth so that discord cant protect it
or something like that idrk

**Comment by Dxniel0** [comment]
What is the situation with the bot?

**Comment by phoebe-leong** [comment]
im not sure. theblockbuster1 said that hed get around to it when he had the time. if need be i can take his advice and fork it and try to do it myself.
im not very good with javascript, but i think it would be good to learn.

**Comment by Washong** [comment]
can somebody close this for inactivity please?
